src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -53,6 +55,16 @@ public BlobProcessorTask(
         super(cloudBlobClient, documentProcessor, envelopeProcessor, errorWrapper);
     }
 
+    /**
+     * Spring overrides the {@code @Lookup} method and returns an instance of bean.
+     *
+     * @return Instance of {@code ServiceBusHelper}
+     */
+    @Lookup
+    public ServiceBusHelper serviceBusHelper() {
+        return null;
+    }
```

Comment:
can we inject this dependency the same way as all the other ones? (via constructor)

Reply:
This way a new service bus client is created for every run of the task. If we inject it like the other dependencies there will only be a single client for the entire lifetime of the application. For the time being I think this is safer as the client gets renewed.

Reply:
I agree - used to have a problem with the client not recovering after a broken connection.